### PR TITLE
e2e: add rest client pkg

### PR DESCRIPTION
**Pull request by filariow**
Part of #126

Signed-off-by: Francesco Ilario <filario@redhat.com>

### CODE REVIEW

File: e2e/pkg/rest/client.go [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+func BuildRESTMapper() (meta.RESTMapper, error) {
+	apiConfig, err := clientcmd.NewDefaultClientConfigLoadingRules().Load()
+	if err != nil {
+		return nil, fmt.Errorf("error building config: %v", err)
+	}
+
+	cfg, err := clientcmd.NewDefaultClientConfig(*apiConfig, &clientcmd.ConfigOverrides{}).ClientConfig()
+	if err != nil {
+		return nil, fmt.Errorf("error building config: %v", err)
+	}
```
Comment (sadlerap) [inline e2e/pkg/rest/client.go:64]: For a future PR: we should extract this and the config stuff under `e2e/hook/` to common code.

**Review by sadlerap — APPROVED** [review]

File: e2e/pkg/rest/client.go (hunk 1 above)
Comment (filariow) [reply e2e/pkg/rest/client.go:64]: that's partially done in this PR and #126

Comment (filariow) [reply e2e/pkg/rest/client.go:64]: sorry, I just got it. Yeah, that's something we should definitely do in a follow-up PR! 😄

**Review by sadlerap — APPROVED** [review]

**Comment by sadlerap** [comment]
lgtm, please rebase/resolve merge conflicts

**Review by sadlerap — APPROVED** [review]

**Pull request merged**